arm-servicebus/2014-09-01/swagger/servicebus.json

```diff
@@ -0,0 +1,974 @@
+        }
+      },
+      "required": [
+        "name"
+      ],
+      "description": "Parameters supplied to the Check Name Availability for Namespace and ServiceBus."
+    },
+    "CheckAvailabilityResource": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Gets or sets the id"
+        },
+        "location": {
+          "type": "string",
+          "description": "Gets or sets datacenter location"
+        },
+        "name": {
+          "type": "string",
+          "description": "Gets or sets name"
+        },
+        "type": {
+          "type": "string",
+          "description": "Gets or sets resource type"
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Gets or sets tags"
+        },
+        "isAvailiable": {
+          "type": "boolean",
+          "description": "Gets or sets true if the name is available and can be used to create new Namespace/ServiceBus. Otherwise false."
+        }
+      },
+      "description": "Description of a CheckAvailibility resource."
+    },
+    "NamespaceProperties": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The name of the namespace."
+        },
+        "provisioningState": {
+          "type": "string",
+          "description": "Gets or sets provisioning state of the Namespace."
+        },
+        "region": {
+          "externalDocs": {
+            "url": ""
+          },
+          "type": "string",
+          "description": "Specifies the targeted region in which the namespace should be created. It can be any of the following values: Australia EastAustralia SoutheastCentral USEast USEast US 2West USNorth Central USSouth Central USEast AsiaSoutheast AsiaBrazil SouthJapan EastJapan WestNorth EuropeWest Europe"
+        },
+        "status": {
+          "externalDocs": {
+            "url": ""
+          },
+          "type": "string",
+          "description": "Status of the namespace. It can be any of these values:1 = Created/Active2 = Creating3 = Suspended4 = Deleting"
+        },
+        "createdAt": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The time the namespace was created."
+        },
+        "serviceBusEndpoint": {
+          "externalDocs": {
+            "url": ""
+          },
+          "type": "string",
+          "description": "Endpoint you can use to perform ServiceBus operations."
+        },
+        "subscriptionId": {
+          "type": "string",
+          "description": "The Id of the Azure subscription associated with the namespace."
+        },
+        "scaleUnit": {
+          "type": "string",
+          "description": "ScaleUnit where the namespace gets created"
```

Comment:
What does the scaleUnit mean in this context? The description is unclear. Can you provide more info or a link from where the user can find more info about scaleUnit?